lib/api/objectPut.js

```diff
@@ -134,7 +137,8 @@ function objectPut(authInfo, request, streamingV4Params, log, callback) {
                 oldByteLength: isVersionedObj ? null : oldByteLength,
             });
             monitoring.promMetrics('PUT', bucketName, '200',
-                'putObject', newByteLength, oldByteLength, isVersionedObj);
+                'putObject', newByteLength, oldByteLength, isVersionedObj,
+                undefined, isIngest);
```

Comment:
I think it's generally best practice, if you're setting the value, to use `null` over `undefined`

Reply:
Ahh okay, I'll change it. Thank you